misc/run-command-shim

```diff
@@ -4,7 +4,7 @@ set -euo pipefail
 readonly SCRIPT_DIR=$(dirname "$0")
 readonly LOG_DIR="/var/log/azure/run-command"
 readonly LOG_FILE=handler.log
-readonly ARCHITECTURE=$(uname -p)
+readonly ARCHITECTURE=$( [[ "$(uname -p)" == "unknown" ]] && echo "$(uname -m)" || echo "$(uname -p)" ) #ternary operator
```

Comment:
quick question for my knowledge. How do I access test coverage for the changes for this repo ?